security.php

```diff
@@ -155,23 +155,23 @@ function wpcom_vip_lost_password_limit( $errors ) {
 function wpcom_vip_username_is_limited( $username, $cache_group ) {
 	$ip = preg_replace( '/[^0-9a-fA-F:., ]/', '', $_SERVER['REMOTE_ADDR'] );
 
-	$key1                   = $ip . '|' . $username;
-	$key2                   = $ip;
-	$count1                 = wp_cache_get( $key1, $cache_group );
-	$is_restricted_username = wpcom_vip_is_restricted_username( $username );
-	$threshold2             = 50;
+	$key1 = $ip . '|' . $username;
+	$key2 = $ip;
+	
+	$threshold1 = apply_filters( 'wpcom_vip_ip_username_login_threshold', 5 );
+	$threshold2 = apply_filters( 'wpcom_vip_ip_login_threshold', 50 );
```

Comment:
Might be useful to some clients to make the `$ip` and the requested host / URL available in the filter, so that they could set values based on requests from an IP range, or for particular subsites in a multisite install.

Reply:
Yeah, that would be great.

```
$ip = preg_replace( '/[^0-9a-fA-F:., ]/', '', $_SERVER['REMOTE_ADDR'] );
```

Unfortunately this is probably not that useful to just pass as an argument to the filter. Happy to do whatever makes sense though.

> or for particular subsites in a multisite install.

I'm thinking it may be better to just run `get_current_blog_id()` for this. Depending on how the multisite is setup, it may or may not matter what the blog ID is. (For subdirectory installs, logging in to any site logs you into every site.) Because of that, I think it could be misleading to pass the blog ID here. Does that seem reasonable?

Reply:
Actually, reading that regex again, it's just removing invalid characters. I'll pass the IP to the filters.